config/credentials.yml.enc

```diff
@@ -1 +1 @@
\ No newline at end of file
+KIb4Xf3Pnm63UQPHb36Shg3WyO5m/5iDyKOq8ZBGVgAHc1T4jtSIUQD8QTSLviTgcJzP9gHo83Mtwwt5gFx4BwqSrXSDPdaRw55k0XfyGdr1VTphArfIB6guN7ATQnlXVAFtGdKgmTdGyN8JpEfqdyLUFqwvwiEBFPeMp77ZbAFrLlA5tED7JjJGXv+rMgjVz0rjqjrE99R5fqz+IFufkpp/BZRq+Ocdga+B2hX3NF0edC1/LJVKZ6hVRWxDwMDeepfVkQIW0UI+d1eTLcG1b9izWqG/P6Ul6NEYvrADyl9ahotHcGbLVxvzGPIKDw4vJXIEtn9ATS380Zr+Jg2/9SQIwWh9SeEHBWqp0QmMwqqeJEequSjX3EhWFtcCStvr2OUMRnO5iTA/nocEeFT+Cq2Ozk8vbukUd1yFwD1lGPQulDFEfxTlMom9WX5aXneErEAm1vjH9b20JkwHpfDm6iwlN7kn22uzUJeLHVZhVrIvWoJ/1l/hckSLaHf4sJLnoE4MxNo+dDZB5oHNQ6IjNeoQlxWmRdHJMy/slVPx0VhwhcDuXu2Se4cDSZ4/vUZec/FpcJgosRsx8wZrVaYuXoXFKuvT29v8N848AT6jATeMZQSSmSdB7xmIaCpcC3dJ3fny8wPoomm3DXLvClDtX30+YY5pxFdTfbxwZ6p8ZGs3NmhYoXKWnSdQF3bl8SnnBC8vuJkPD3bQkStMT3npIPMvKc/jujnkeEEakSEdT+BDkoYu+8EbJlsG5PiSKeIde/bjalWjoSvv4f22Hep0PnpbbLuHc+gtDbNzjDhcH04KQQmsVHxxjQQJTRRmvD6SEyJfth1fJqAOVgxX+V4cdeXSYfxiHIDgV9de/W01f175XBB7nuFQQFS16msHO0QT1FQnLFIe218+XbahOihL8jAIz+CyeQjdcBvRFIZGTbWMbXZXp+Sr6z1ZdYvDNmwUqjHRnuVJ6U3P5DDG/X6DF3a/Zfq8OSJT+alBr+4QyHGL51y6yCw+7WuYbbP8+17hCE7gDYJwKLWrgF40KFcAGVgqUYyd5+JGPqowRYugQPGHYeui864vKkdXy1B4NgOudT/QxWxlZZDSV/76neszGbHsjcsiC80WSfkzHW9FSB0ZC9nT+e2x/XiO0JvUyraWtKGz7UL63GNKbNLrUxYz/wnXe6ERW6/yu3j6wqI5sqHNhyvaQWbgCkT19VzcKrFqzt/lwemaXolqGrTdbcoJGm0MrRqrKtJZzbT5ypD3AiXmgdI8YyoGTA2UBKqk6DKzwSF/GdHacK4UG2T9t5WXq/PcClgOsHRI0J35GiOeBozV7vFFiL1Tbuqr3k33qlYvLOqByOYpzbNk5OqWsIUBpg7XczXXBNcWtR9HiuITyPF1qd7xP1PvenT4Tcy2D2QshK+m6SSHV+aZcd1lw4NU9tswGdsgmr26+4rX/UDYN/4owLs7KD61qsRuNhtmPxplRG/1dvxkz8eIJrC3IQxC0pvInXTc2uylHLsAQror2Jg7xraWr1m3jWJ/Q/bOqQrmrX5cp6xx2MGG5w2V6MkTl/wC/ZpQF2MSqme71trahFY9M31m4I3PM7qppNU=--saBqjftGBdPoAqJf--KRUaoeAk4M02/FhwuonT3Q==
```

Comment:
⚠️ we will need to add the `mainserver.url` for alle releases in the releases branch and in the configs of all saas applications

Reply:
You mean here?

![Bildschirmfoto 2023-05-08 um 09 37 55](https://user-images.githubusercontent.com/90779/236764457-67ccd0f2-b996-4cf0-9eca-f7686e4d3166.png)

Reply:
⚠️ this is not the case anymore!!!